app/screens/NewScorecard/NewScorecard.js

```diff
@@ -80,9 +80,20 @@ class NewScorecard extends Component {
     });
   }
 
+  isScorecardAlreadySaved = () => {
```

Comment:
@limkimsan I would suggest moving this method into scorecard_service instead with isExists(code) {
  return find(code) != undefined
}

Please consider moving every codebase that touches the realm object into service where every view component should call from the service.

Reply:
Bong, I have moved methods that contain realm to service file.